test/statements/bharshbarg/breakOutsideOfLoop.good

```diff
@@ -1,2 +1,3 @@
 breakOutsideOfLoop.chpl:1: In function 'main':
-breakOutsideOfLoop.chpl:2: error: break or continue is not in a loop
+breakOutsideOfLoop.chpl:2: error: 'break' is not allowed outside of a loop
+breakOutsideOfLoop.chpl:1: note: stopped searching here because 'break' statements are not allowed to jump outside a procedure
```

Comment:
The ``'break' is not allowed outside of a loop`` text seems sufficient here to me.